docs/concepts/clustering.rst

```diff
@@ -1,3 +1,4 @@
+.. _concept-clustering:
 .. _clustering:
```

Comment:
Isn't one reference enough?

Reply:
I didn't want to break old references. But I am also happy to go forward and just break them, accompanied by fixing them again. However, there might be some delay due to different release cycles, so I was hesitant on this.

You decide.

Reply:
Imho let's give it one reference and fix it downstream ASAP, unless you anticipate the delay to be quite long

Reply:
I think we should avoid a situation where we maintain dozen of reference labels for BWC for content that we can control anyway.
But I think we should get the input of @norosa on how we should handle this.

Documentation usually has no release cycle, but instead the latest state on a branch is always directly made available.

Do we have link checks in all the repositories in place that use the cross-referencing feature so that we notice if we break anything?
We probably need to set up some periodic checks for all of it so that we notice if we break anything.

Reply:
With e482d34a, I purged all duplicate references and adjusted everything within this space. Links from downstream repository will just follow as we go.

Reply:
> Do we have link checks in all the repositories in place that use the cross-referencing feature so that we notice if we break anything?

Mostly, I believe. Where I find them missing, I am adding it.

> We probably need to set up some periodic checks for all of it so that we notice if we break anything.

Sure, we can do that.